### PR TITLE
Query multiple tables

### PR DESCRIPTION
**Pull request by joel-kalema**
- [ ] Create a table named owners with the following columns:
      - id: integer (set it as autoincremented PRIMARY KEY)
      - full_name: string
      - age: integer
- [ ] Create a table named species with the following columns:
      - id: integer (set it as autoincremented PRIMARY KEY)
      - name: string
- [ ] Modify animals table:
      - Make sure that id is set as autoincremented PRIMARY KEY
      - Remove column species
      - Add column `species_id` which is a foreign key referencing species table
      - Add column `owner_id` which is a foreign key referencing the owners table
- [ ] Remember all this goes in `schema.sql`.
- [ ] Insert the following data into the owners table:
- [ ] Insert the following data into the `species` table:
       - Pokemon
       - Digimon
- [ ] Modify your inserted animals so it includes the species_id value
- [ ] Modify your inserted animals to include owner information (owner_id) 
- [ ] Remember these insertions and modifications happen in `data.sql`.
- [ ] Write queries (using JOIN) to answer the following questions.

### CODE REVIEW

File: queries.sql [hunk 1]
```diff
@@ -1,70 +1,47 @@
...
+  -- Sam Smith owns Agumon.
+  UPDATE animals SET owner_id = 1 WHERE name = 'Agumon';
+  -- Jennifer Orwell owns Gabumon and Pikachu.
+  UPDATE animals SET owner_id = 2 WHERE name = 'Pikachu' OR name = 'Gabumon';
+  -- Bob owns Devimon and Plantmon.
+  UPDATE animals SET owner_id = 6 WHERE name = 'Devimon' OR name = 'Plantmon';
+  -- Melody Pond owns Charmander, Squirtle, and Blossom.
+  UPDATE animals SET owner_id = 3 WHERE name = 'Squirtle' OR name = 'Blossom' OR name = 'Charmander';
+  -- Dean Winchester owns Angemon and Boarmon.
+  UPDATE animals SET owner_id = 4 WHERE name = 'Angemon' OR name = 'Boarmon';
```
Comment (keroloussamy) [inline queries.sql]: - [ ] Kindly in your `queries.sql` file, you should not insert data, you should write your queries.
- [ ] Kindly you have to write these queries, as asked by the requirements in your `queries.sql` file. Now I don't see any of them.
image

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @joel-kalema, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯


- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: queries.sql (hunk 1 above)
Comment (joel-kalema) [reply queries.sql]: ohhh, sorry 
I added the bad file.

**Review by brytebee — APPROVED** [review]
Hi, @joel-kalema👋,

# STATUS: APPROVED 🕺🥳 

Congratulations🍾🥳!! Your project is ready for presentation!  There is nothing else to say other than... it's time to merge it :shipit: 




Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @brytebee in your question so I can receive the notification.
### _**Use @brytebee on slack for faster responses.**_

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**